darts/timeseries.py

```diff
@@ -705,6 +730,106 @@ def update(self,
             new_series = new_series.astype(self._df.dtypes)
         return TimeSeries(new_series, self.freq())
 
+    def stack(self, other: 'TimeSeries') -> 'TimeSeries':
+        """
+        Stacks another univariate or multivariate TimeSeries with the same index on top of
```

Comment:
Details but I would maybe clarify in doc that stacking is not just adding values to each other and creating one timeseries but more like adding a new component.

Reply:
done

Reply:
done